packages/frontend/src/contexts/info.js

```diff
@@ -39,4 +41,8 @@ export default class Info {
     this.EPOCH_TREE_ARITY = EPOCH_TREE_ARITY
     this.EPOCH_KEY_NONCE_COUNT = EPOCH_KEY_NONCE_COUNT
   }
```

Comment:
the home page is not displaying the unirep contract address (link) or the contract config info. Can we get these from the Graph as well?

Reply:
If you start the backend, you will see the unirep contract address and the contract config info. But it's a good question that should we put it else where, e.g. in an .env file?

Reply:
ok yeah thx i see now. but i'm confused about a few things:

the subgraph is replacing the functionality of the synchronizer, correct? then it seems that getting this contract info is currently the synchronizer's only job?

are the attesters on goeril, mumbai, and sepolia fake for now or do we actually have contracts deployed on the networks? bc currently we are providing the arbitrum/goerli address and link to arbiscan in the info box for all networks.

Reply:
yeah we deployed on goeril, mumbai, and sepolia 
arbitrum-goerli: 0xCa61bFcA0107c5952f8bf59f4D510d111cbcE146
goerli: 0xCa61bFcA0107c5952f8bf59f4D510d111cbcE146
mumbai: 0xCa61bFcA0107c5952f8bf59f4D510d111cbcE146
sepolia: 0xCa61bFcA0107c5952f8bf59f4D510d111cbcE146

Reply:
oh i didn't know you could deploy with the same address! so should the address links go to the different networks' explorers or all to arbiscan?

Reply:
yeah they should like to different explorers
e.g.
sepolia: https://sepolia.etherscan.io/
goerli: https://goerli.etherscan.io/
mumbai: https://mumbai.polygonscan.com/
arbitrum-goerli: https://goerli.arbiscan.io/